Source/WTF/wtf/JSONValues.cpp

```diff
@@ -492,16 +525,55 @@ RefPtr<Value> Value::asValue()
 
 RefPtr<Object> Value::asObject()
 {
```

Comment:
Now that these DJ ctions are so super-simple, maybe they should be inlined?

Reply:
Yeah, changed.